### PR TITLE
For unauthorized user send isAuthenticated false

### PR DESCRIPTION
**Pull request by nil1511**

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ apiRouter.get('/verify', (req, res) => {
   }, process.env.JWT_SECRET, {
     expiresIn: '1 day',
   });
-  if (req.username) {
+  if (req.username && req.permissions && req.permissions.length) {
```
Comment (adcpm) [inline routes/api.js]: It should not be based on permissions. Some app may be authorised without having any posting permission. We should check if user approved the app.

Comment (nil1511) [inline routes/api.js]: That would rely on db check. also all permissions need posting wif access and if non of them are checked. shouldn't it be unauthorised?

Comment (adcpm) [inline routes/api.js]: If nothing is checked and user approve then its authorised. Some app just need to verify user identity but they dont need to broadcast operation (for example: a chat).

Comment (nil1511) [inline routes/api.js]: Now it should work accordingly.

Comment (nil1511) [inline routes/api.js]: Now it should work accordingly.

**Pull request merged**